packages/common/src/chains/index.ts

```diff
@@ -0,0 +1,5 @@
+// re-export wagmi chains to make them easier to reach
+export * from "@wagmi/chains";
```

Comment:
in the future, we might consider either not doing this (users include `@wagmi/chains` in their own deps) or make it a peer dep so that users can upgrade the chains version to get new chains without us having to constantly chase version updates here

Reply:
nevermind, gonna just remove the re-export now